src/components/Community/Button/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { Wrapper } from './styles'
+
+function BareButton({ theme, children, forwardedRef, ...props }) {
+  return (
+    <Wrapper {...props} {...theme} ref={forwardedRef}>
+      {children}
+    </Wrapper>
+  )
+}
+
+BareButton.propTypes = {
+  children: PropTypes.node,
+  theme: PropTypes.shape({
+    backgroundColor: PropTypes.string,
+    color: PropTypes.string
+  }),
+  forwardedRef: PropTypes.func
+}
+
+// eslint-disable-next-line react/display-name
+export default React.forwardRef((props, ref) => (
+  <BareButton {...props} forwardedRef={ref} />
+))
```

Comment:
I think you can make the BareButton a wrappee of `React.forwardRef` itself, instead of adding an extra component around it. It should work fine even if you add hooks to the component eventually.

Reply:
It's mostly done this way to let the button have a readable name to show in dev tools, bow I start to think that I can do both without separate component, but I need to experiment some more.

Reply:
Tried, didn't work (you can't default export consts, only functions). But I renamed BareButton to CommunityButton for consistency.